### DISCUSSION TITLE
[BUG] Music -> Mute ambient sounds

### DISCUSSION THREAD

**Discussion by swiggly** [runelite/runelite#17177] [Ideas]
"Mute Ambient Sounds" does not stay active. Will unmute after a period of time and the toggle does not change.

Have to untoggle and retoggle "Mute ambient sounds" to make the sounds go away.

IE: The ambient noises by the magic trees in the Woodcutting guild.

**Comment by Adam-** [comment]
This is a bug introduced by the latest game update. It has been fixed in c2fe2958837ca7407d4730bb9ca5439458621ff2 which will be released likely tomorrow.
